internal/cli/server/config.go

```diff
@@ -406,27 +409,28 @@ func DefaultConfig() *Config {
 			URL:     "http://localhost:1317",
 			Without: false,
 		},
-		SyncMode: "full",
-		GcMode:   "full",
-		Snapshot: true,
+		SyncMode:   "full",
+		GcMode:     "full",
+		Snapshot:   true,
+		NoSnapshot: false,
 		TxPool: &TxPoolConfig{
 			Locals:       []string{},
 			NoLocals:     false,
 			Journal:      "",
 			Rejournal:    time.Duration(1 * time.Hour),
-			PriceLimit:   1,
+			PriceLimit:   30000000000,
 			PriceBump:    10,
 			AccountSlots: 16,
-			GlobalSlots:  4096,
-			AccountQueue: 64,
-			GlobalQueue:  1024,
+			GlobalSlots:  32768,
```

Comment:
When did we decide to change these values? Was this changed before in Bor?

Reply:
These values are currently being used in bor nodes. I think this change was when we faced an issue with the alchemy node with the transactions. We've been using this since then. 

@ssandeep can add more info here!